keyboards/sneakbox/aliceclone/kb.h

```diff
@@ -0,0 +1,30 @@
+#ifndef KB_H
```

Comment:
File should be renamed to `aliceclone.h`

Reply:
Got it. Changed filename to aliceclone.h